mobile/src/bundles/common/components/autocomplete-selectors/autocomplete-selector.tsx

```diff
@@ -39,12 +47,13 @@ const AutocompleteSelector = <T extends FieldValues>({
     placeholder,
 }: Properties<T>): JSX.Element => {
     const { field } = useFormController({ name, control });
-    const { onBlur, onChange } = field;
+    const { value, onBlur, onChange } = field;
     const [search, setSearch] = useState('');
     const { isVisible, toggleVisibility } = useVisibility(false);
 
```

Comment:
I needed to return the value from the field and return the arrow icon, otherwise this field on the form is counted incorrectly and does not match the design.